Source/i_video.c

```diff
@@ -28,6 +28,7 @@
 
 #include "SDL.h" // haleyjd
 
+#include "config.h"
```

Comment:
Same here. `i_video.c` already includes `doomstat.h` which by some other dependency chain already includes `config.h`. BTW, do we define `HAVE_CONFIG_H` anywhere in the CMake build?

Reply:
Rats, I did not notice that define.  That would explain the errors I was running into.

There are two ways forward.  I can either hard-code that define as a compiler flag, or we can get rid of the define completely, since CMake stamps out `config.h` in all cases.

I would prefer the latter, as it simplifies things in the absence of alternative build systems, but the choice is ultimately yours.